risc0/r0vm/src/lib.rs

```diff
@@ -163,6 +170,7 @@ impl Cli {
         };
         let opts = ProverOpts {
             hashfn: hashfn.to_string(),
+            prove_guest_errors: false,
```

Comment:
I feel like maybe we should punt on this feature, unless it's really necessary for proof composition.

Reply:
So mostly this is useful to separate 'proof is invalid' from 'guest returned an error', which right now is mostly about panics, but also sets us up well for more general proof-of-fault support.  I'm not sure if we ended up deciding that proof-of-fault was a 1.0 feature, but even if it's not, this feature will reduce eventual API changes stemming from proof-of-fault, so given that it's already implemented, I think it's a good thing to keep.